src/main/java/org/jenkinsci/plugins/github_branch_source/Connector.java

```diff
@@ -102,11 +102,6 @@
 
         gb.withRateLimitHandler(CUSTOMIZED);
         OkHttpClient client = new OkHttpClient().setProxy(getProxy(defaultIfBlank(apiUrl, GITHUB_URL)));
-        client.setCache(GitHubClientCacheOps.toCacheDir().apply(config));
-        if (config.getClientCacheSize() > 0) {
-            Cache cache = toCacheDir().apply(config);
-            client.setCache(cache);
-        }
```

Comment:
@jglick We can find scenarios where they do not use webhooks but I hope you agree with me that one important feature of this plugin is just to build _stuff_ when _something_ happens in the source code. In this context, `polling` is not the most appropiate procedure.

I agree with remove the cache because the most important information from GitHub is needed in _hot_, without client cache.